Makefile

```diff
@@ -25,10 +25,6 @@ build: format
 	go mod tidy
 	go build $(LDFLAGS)
 
-security-scan:
-	go get github.com/securego/gosec/cmd/gosec
-	gosec -no-fail ./...
-
```

Comment:
Small doubt that I have here, why are we deleting this security scan?

Reply:
@w0rmr1d3r this should be added back, I need to do some more experimenting with getting it working. Right now gosec is scanning the `.github/workflows` dir

Reply:
Perfect! Thank you!